lib/features/home_view/widgets/buildings_section/buildings_section.dart

```diff
@@ -73,7 +70,7 @@ class _DataListBuildingsTiles extends ConsumerWidget {
             buildingName: mapItem.name,
             imageUrl: mapItem.cover?.filename_disk?.directusUrl,
             onTap: () async {
-              await BuildingsSection.goToMapTab(ref);
+              unawaited(ref.navigateBuildings());
```

Comment:
`unawaited `here's very very nice since we don't use the Future

Reply:
Linter screams if you don't do it :) now you have to either await the future or unwait